app/celery/scheduled_tasks.py

```diff
@@ -111,21 +111,13 @@ def delete_invitations():
 @notify_celery.task(name='timeout-sending-notifications')
 @statsd(namespace="tasks")
 def timeout_notifications():
-    # TODO: optimize the query by adding the date where clause to this query.
-    notifications = get_notifications(filter_dict={'status': 'sending'})
-    now = datetime.utcnow()
-    for noti in notifications:
-        try:
-            if (now - noti.created_at) > timedelta(
-                    seconds=current_app.config.get('SENDING_NOTIFICATIONS_TIMEOUT_PERIOD')
-            ):
-                # TODO: think about making this a bulk update rather than one at a time.
-                updated = update_notification_status_by_id(noti.id, 'temporary-failure')
-                if updated:
-                    current_app.logger.info(
-                        "Timeout period reached for notification ({}), status has been updated.".format(noti.id))
-        except Exception as e:
-            current_app.logger.exception(e)
-            current_app.logger.error(
-                "Exception raised trying to timeout notification ({}) skipping notification update.".format(noti.id)
-            )
+    try:
+        updated = dao_timeout_notifications(current_app.config.get('SENDING_NOTIFICATIONS_TIMEOUT_PERIOD'))
+        if updated:
+            current_app.logger.info(
+                "Timeout period reached for {} notifications, status has been updated.".format(updated))
+    except Exception as e:
+        current_app.logger.exception(e)
+        current_app.logger.error(
+            "Exception raised trying to timeout notification skipping notification update."
+        )
```

Comment:
dumb q - do we gain anything from this exception? before it was making sure it tried every notification but celery will still log task as failed if it just bubbles up - i'd just kill this try except block